lambdas/utils/common_query_filters.py

```diff
@@ -0,0 +1,29 @@
+from enums.dynamo_filter import AttributeOperator, ConditionOperator
+from enums.metadata_field_names import DocumentReferenceMetadataFields
+from utils.dynamo_query_filter_builder import DynamoQueryFilterBuilder
+
+NotDeleted = (
+    DynamoQueryFilterBuilder()
+    .add_condition(
+        attribute=str(DocumentReferenceMetadataFields.DELETED.value),
+        attr_operator=AttributeOperator.EQUAL,
+        filter_value="",
+    )
+    .build()
+)
+
+UploadCompleted = (
+    DynamoQueryFilterBuilder()
+    .add_condition(
+        attribute=str(DocumentReferenceMetadataFields.DELETED.value),
+        attr_operator=AttributeOperator.EQUAL,
+        filter_value="",
+    )
+    .add_condition(
+        attribute=str(DocumentReferenceMetadataFields.UPLOADED.value),
+        attr_operator=AttributeOperator.EQUAL,
+        filter_value=True,
+    )
+    .set_combination_operator(ConditionOperator.AND)
```

Comment:
This is a really nice addition to the filter builder, I think Noga added a filter to dynamo_utils.py called `filter_expression_for_available_docs()`, is there any way we can bring that into this file similar to how you've created these common filters for consistency? 

Also the ConditionOperator is set to AND by default so this line isn't necessary unless you were changing to another Operator, but I guess for readability it is more clear to be specific and can be kept in so will leave it up to you

Reply:
Also for this filter do we want to check that Uploaded = True and also Uploading = False? or is this a scenario that cannot happen?

Reply:
Thanks!  The `UploadCompleted` filter was just a placeholder to avoid InternalServerError. 
When I merged with Noga's branch just now, I found that the filter logic is different, so I will prefer `filter_expression_for_available_docs` over UploadCompleted for now (so that I don't accidentally mess up with the logic in that branch)
Later after the story is tested working, we can move filter_expression_for_available_docs() into a common filter AvailableDocs here